src/CAN/CANMotor.cpp

```diff
@@ -0,0 +1,100 @@
+#include "CANMotor.h"
+
+#include "CAN.h"
+#include "CANUtils.h"
+
+#include <chrono>
+#include <cmath>
+#include <thread>
+
+extern "C" {
+#include "../HindsightCAN/CANCommon.h"
+#include "../HindsightCAN/CANMotorUnit.h"
+#include "../HindsightCAN/CANPacket.h"
+}
+
+using namespace std::chrono_literals;
+using robot::types::DataPoint;
+using robot::types::LimitSwitchData;
+
+namespace can::motor {
+
+void emergencyStopMotors() {
+	CANPacket p;
+	AssembleGroupBroadcastingEmergencyStopPacket(
+		&p, static_cast<uint8_t>(devicegroup_t::motor), ESTOP_ERR_GENERAL);
+	can::sendCANPacket(p);
+	std::this_thread::sleep_for(1000us);
```

Comment:
Are these waiting times to give the CAN buffer a chance to empty out? I think I remember them being in the original code

Reply:
Yeah, I'm not sure if it's needed or not, but that's what the old code did. We could try to find out the size of the packet buffer on the motor boards, and if it's large enough we could probably remove the sleeps?